arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -86,21 +80,14 @@ class EditAndSyncFeaturesViewController: UIViewController {
                 self.presentAlert(error: error)
             } else {
                 let featureServiceInfo = self.geodatabaseSyncTask.featureServiceInfo!
-                let map = self.mapView.map!
-                for index in featureServiceInfo.layerInfos.indices {
+                for index in featureServiceInfo.layerInfos.indices.reversed() {
                     // For each layer in the serice, add a layer to the map.
                     let layerURL = self.featureServiceURL.appendingPathComponent(String(index))
                     let featureTable = AGSServiceFeatureTable(url: layerURL)
-                    featureTable.load { [weak self, unowned featureTable] error in
```

Comment:
No need to explicitly load the feature layers, and resolved the wrong reference count issue.